tilerator/data_tm2source.xml

```diff
@@ -0,0 +1,332 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE Map[]>
+<Map srs="+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0.0 +k=1.0 +units=m +nadgrids=@null +wktext +no_defs +over">
+
+<Parameters>
+  <Parameter name="attribution"><![CDATA[<a href="http://www.openmaptiles.org/" target="_blank">&copy; OpenMapTiles</a> <a href="http://www.openstreetmap.org/about/" target="_blank">&copy; OpenStreetMap contributors</a>]]></Parameter>
+  <Parameter name="center">-12.2168,28.6135,4</Parameter>
+  <Parameter name="description"><![CDATA[A tileset showcasing all layers in OpenMapTiles. http://openmaptiles.org]]></Parameter>
+  <Parameter name="maxzoom">14</Parameter>
+  <Parameter name="minzoom">0</Parameter>
```

Comment:
These values may have unexpected effects (e.g on raster tiles).
Should we set `maxzoom` to a higher value ?

Reply:
I would trust you on this 😜 

what `maxzoom` do you think would be best ?